akka-actor/src/main/mima-filters/2.5.x.backwards.excludes

```diff
@@ -4,3 +4,24 @@ ProblemFilters.exclude[MissingClassProblem]("akka.actor.ActorDSL$")
 ProblemFilters.exclude[MissingClassProblem]("akka.actor.ActorDSL")
 ProblemFilters.exclude[MissingClassProblem]("akka.actor.ActorDSL$*")
 ProblemFilters.exclude[MissingClassProblem]("akka.actor.dsl.*")
+
+# #26190 remove actorFor
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.actor.ActorCell.actorFor")
```

Comment:
why is each one duplicated?

Reply:
don't know, maybe it was several warnings and I just copied - I'll remove the duplicates